coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/RelToTrinoConverter.java

```diff
@@ -89,6 +107,15 @@ public SqlNode convertToSqlNode(RelNode relNode) {
     return visitChild(0, relNode).asStatement();
   }
 
+  /**
+   * Convert input relational algebra to CoralSqlNode
+   * @param relNode relation algebra
+   * @return CoralSqlNode representation for input
+   */
+  public SqlNode convertToCoralSqlNode(RelNode relNode) {
```

Comment:
I don't think we need this explicit method given it's simple and only called in `convertDash`.